src/bin/pgcopydb/schema.c

```diff
@@ -3280,7 +3280,7 @@ schema_list_partitions(PGSQL *pgsql,
 	int64_t max = table->partmax;
 
 	int64_t partsCount = 1;
-	int64_t partsSize = max - min + 1;
+	int64_t partsSize;
```

Comment:
I do not think this improves code clarity / readability, even though I understand why it is “correct”.

Reply:
It is apparently redundant and thus increases the cognitive overhead IMHO.